app/views/pages/developer-guide/apis/creating-items-from-external-api-call.liquid

```diff
@@ -36,44 +48,78 @@ properties:
   type: integer
 ```
 
-### Step 2: Create Form
+### Step 3: Create a REST endpoint to store a new flower
 
-Define what data you expect, and required validations:
+Create GraphQL mutations to store your data. Place these GraphQL files in the `app/graphql/` folder.
 
-##### app/forms/import_flowers.liquid
+To add a new flower, create a new file `app/graphql/books/create_flower.graphql`:
 
-```yaml
+##### app/graphql/books/create_flower.graphql
+
+```graphql
+mutation($name: String!, $creator_id: Int!) {
+  flower: model_create(
+    model: {
+      model_schema_name: "book"
+      properties: [
+        { name: "name", value: $name }
+        { name: "creator_id", value_int: $creator_id }
+      ]
+    }
+  ) {
+    id
+  }
+}
+```
+
+The next step is to create a new page used as a REST endpoint for your web application. We recommend storing all endpoints used in the API inside the `app/views/pages/api` folder to ensure separation from other site sections.
+
+Additionally, when planning to use different request methods for one URL like `POST /api/flowers`, `GET /api/flowers`, etc., we suggest naming the files after the request method related to the given endpoint. Place the endpoint responsible for adding new books in `app/views/pages/api/flowers/post.liquid`.
+
+##### app/views/pages/api/flowers/post.liquid
+
+```liquid
+{% raw %}
 ---
-name: import_flowers
-resource: flower
-fields:
-  properties:
-    name:
-      validation:
-        presence: true
-    creator_id:
-      validation:
-        presence: true
-resource_owner: anyone
+slug: api/flowers
+method: post
+layout: ''
+response_headers: >
+  {% render 'shared/api/headers' %}
 ---
+
+{% liquid
+  graphql result = 'flowers/create_flower', name: context.params.name, creator_id: context.params.creator_id
+  unless result.flower
+    response_status 400
+  endunless
+%}
+
+{{ result }}
+{% endraw %}
 ```
 
-### Step 3: Generate API token
+You’ve defined the endpoint’s slug and method and removed the default layout. The previously created `headers.liquid` file is used to render the appropriate response header.
+
+You’re calling the `flowers/create_flower` GraphQL mutation using the request parameters. All GraphQL results always return valid JSON. On error, the `result.flower` key will not be present, and you should add an appropriate error response status code. Finally, render the whole `result` object. It will contain either the new flower id or an error message when creating the flower has failed.
+
+
+### Step 4: Generate API token
 
-Log in to the Partner Portal, and click your user icon on the top right. Select `Your Profile` and click on `Show/Hide` to show the **Access Key**.
+Log in to the Partner Portal, and click your user icon on the top right. Select `Your Profile`, click on `Tokens` and then on `Show/Hide` to show the **Access Key**.
```

Comment:
@Slashek we shouldnt point users to the global tokens that we are deprecating, but to OAuth tokens only. But currently you cant obtain those tokens in PP, only some truncated version. We need to expand this feature.

Reply:
@pavelloz yes, created issue in PP #1150 ; thx!

Reply:
OK, I just marked this issue as `blocked` because we need to update the partner portal first.

Reply:
@Slashek the issue that you linked is a code format fix in the documentation. Could you link the partner portal issue? @diana-lakatos would like to close this one if we don't have any blockers, but I don't know what is the progress of the partner portal feature.

Reply:
I think this is done looking at https://github.com/mdyd-dev/partner-portal/issues/1150
and https://github.com/mdyd-dev/partner-portal/pull/1170 and also "Tokens" under my account settings (ie. https://partners.platformos.com/users/13/access_tokens ) .

In documentation we need to add instruction to show token for particular instance, instead of taking the global token (the one hidden under "Show/hide" button)